lesson16/main.js

```diff
@@ -0,0 +1,119 @@
+"use strict";
+
+const jsonUrl = "https://jsondata.okiba.me/v1/json/BFwf7210926001715";
+const newsWrapper = document.getElementById("js-news");
+const tabUl = document.querySelector(".news__tabs");
+
+const createElementWithClassName = (element, name) => {
+  const createdElement = document.createElement(element);
+  createdElement.classList.add(name);
+  return createdElement;
+};
+
+//comment-icon
+const commentIcon = createElementWithClassName("span", "comment-icon");
+
+//new-label
+const newLabel = createElementWithClassName("span", "new-label");
+newLabel.textContent = "NEW!";
+
+//news-img
+const imgWrapper = createElementWithClassName("div", "news__img");
+
+//tab
+const createNewTab = (data) => {
+  const tabUl = document.querySelector(".news__tabs");
+  const tabFragment = document.createDocumentFragment();
+
+  data.forEach((key,index) => {
+    const tabLi = createElementWithClassName("li", "tab");
+    tabLi.textContent = key.category;
+    tabLi.dataset.tab = key.id;
+    if (index === 0) {
```

Comment:
IMO
you write comment for other developer who don't understand why index of zero is active.

```suggestion
    if (index === 0) { // とりあえず最初のインデックスをアクティブなタブとする
```

Reply:
I added this comment.